packages/eslint-config-udemy-website/index.js

```diff
@@ -37,10 +37,20 @@ module.exports = {
                 source: '^enzyme/build/index(?:\\.js)?$',
                 message: 'Please import from enzyme, not from enzyme/build/index',
             },
+            {
+                // Prefer lodash-es
+                source: '^lodash/(.*)$',
+                message: 'Please import from lodash-es/$1 instead of lodash/$1',
+            },
             {
                 // For improved JS bundling
-                source: '^lodash(?:\\.js)?$',
-                message: 'Please import from e.g. lodash/foo, not from lodash directly',
+                source: '^(lodash(?:-es)?)(?:\\.js)?$',
+                message: 'Please import from e.g. lodash-es/foo, not from $1 directly',
+            },
+            {
+                // Suggest using native Javascript instead of depending on lodash
+                source: '^lodash-es/(each|reduce|filter|includes|map|every|some|keys|values|isArray|isString|findIndex|extend|max|first|head|tail|last)(?:\\.js)?$',
```

Comment:
Note that I am considering separating this list into different rules in the future to have a more helpful message. e.g. use Object.keys instead of `lodash-es/keys`.